src/App.tsx

```diff
@@ -2,21 +2,25 @@ import "./App.css";
 import Form from "./components/Form/Form";
 import Cards from "./components/Cards";
 import { useState } from "react";
-import { ICardData } from "./components/Cards";
+import { AI } from "./components/Form/Form";
 
 function App() {
-  const [formData, setFormData] = useState<ICardData[]>([]);
+  const [formData, setFormData] = useState<AI[]>([]);
 
   const handleDelete = (index: number) => {
     setFormData((prevCards) => prevCards.filter((_, i) => i !== index));
   };
+
+  const handleFormSubmit = (Ai: AI) => {
+    setFormData([...formData, Ai]);
+  };
   return (
     <>
       <main>
         <div className="container">
           <section id="content">
             <div className="main-container">
```

Comment:
please change here html tags, plus where is <ThemeProvider> and custom theme?

Reply:
How to change this? Theme will be added soon

Reply:
simple solution change them to `<Box/> `, this could be helpfull too https://mui.com/material-ui/all-components/#layout